src/blr/cli/bamtofq.py

```diff
@@ -0,0 +1,96 @@
+"Writes fq files from bam files"
+
+import logging
+import pysam
+
+from tqdm import tqdm
+import dnaio
+
+from blr import utils
+
+logger = logging.getLogger(__name__)
+
+
+def main(args):
+    logger.info(f"Starting analysis")
+    query_cache = dict()
+    summary = Summary()
+    out_interleaved = not args.r2_out
+    with pysam.AlignmentFile(args.bam, "rb") as reader, \
+            dnaio.open(args.r1_out, file2=args.r2_out, interleaved=out_interleaved, mode="w",
+                       fileformat="fastq") as writer:
+        for query in tqdm(reader):
+            summary.tot_reads_in += 1
+
+            # Read pair cache system
+            if query.query_name not in query_cache:
+                query_cache[query.query_name] = query
+                continue
+            else:
+                query_1 = query_cache[query.query_name]
+                query_2 = query
+                del query_cache[query.query_name]
+
+            # Fetch and add bam tag to header (if not found header is not modified)
+            if args.tag:
+                add_tag_to_header(query_1, query_2, args.tag)
+
+            # Find out which query is read 1 and read 2 (so they will be output to correct file)
+            r1, r2 = turn_readpair(query_1, query_2)
+
+            r1_as_dnaio_object = dnaio._core.Sequence(name=r1.query_name, sequence=r1.seq, qualities=r1.qual)
+            r2_as_dnaio_object = dnaio._core.Sequence(name=r2.query_name, sequence=r2.seq, qualities=r2.qual)
+
+            writer.write(r1_as_dnaio_object, r2_as_dnaio_object)
+            summary.tot_reads_out += 2
+
+    summary.print_stats()
+    logger.info(f"Finished")
```

Comment:
```suggestion
    logger.info("Finished")
```
This string does not have to be f-string.